src/Minify.php

```diff
@@ -154,6 +154,9 @@ protected function load($data)
         if ($this->canImportFile($data)) {
             $data = file_get_contents($data);
 
+            // replace CR linefeeds etc.
+            $data = preg_replace('~\R~', "\n", $data);
```

Comment:
Hmm sorry. Actually I think the regex is fine now that I had a better look.

I didn't realize it was an uppercase `\R` yesterday. Your current regex seems indeed the most sane one for normalizing CRLF's.